bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PacketProcessorBaseV3.java

```diff
@@ -82,6 +82,9 @@ protected void sendResponse(StatusCode code, Object response, OpStatsLogger stat
                 requestProcessor.getRequestStats().getChannelWriteStats()
                         .registerFailedEvent(MathUtils.elapsedNanos(writeNanos), TimeUnit.NANOSECONDS);
                 statsLogger.registerFailedEvent(MathUtils.elapsedNanos(enqueueNanos), TimeUnit.NANOSECONDS);
+                if (response instanceof BookieProtocol.Response) {
```

Comment:
Doesn't that response is belong to v2 protocol?

Reply:
+1